anoncred-kmm/anoncreds-kmp/src/commonTest/kotlin/io/iohk/atala/prism/anoncred/PrismIssuerTests.kt

```diff
@@ -0,0 +1,226 @@
+package io.iohk.atala.prism.anoncred
+
+import anoncreds_wrapper.AttributeValues
+import anoncreds_wrapper.CredentialDefinitionConfig
+import anoncreds_wrapper.Issuer
+import anoncreds_wrapper.Nonce
+import anoncreds_wrapper.Prover
+import anoncreds_wrapper.RegistryType
+import anoncreds_wrapper.Schema
+import anoncreds_wrapper.SignatureType
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+@AndroidIgnore
+class PrismIssuerTests {
```

Comment:
Ok this is one thing we need to remove if we want to make a PR for the original repo. Remove anything that is `Prism`